.changeset/polite-dodos-happen.md

```diff
@@ -0,0 +1,9 @@
+---
+"miniflare": patch
+---
+
+fix: Allow the magic proxy to proxy objects containing functions indexed by symbols
```

Comment:
Is this description still valid? I don't see much in this PR about symbols. But perhaps I am missing something?

Reply:
https://github.com/cloudflare/workers-sdk/pull/6301#discussion_r1696057576